### PR TITLE
Default account emoji on creation

### PR DESCRIPTION
**Pull request by maksymboytsov**
## Changelog

- Bumped `@psychedelic/plug-controller` to `0.5.2`
- Default emoji adding into PlugWallet on first time account creation
- Chrome sidebar emoji centered

### Type of changes included:

- [ ] Components
- [X] Business Logic
- [X] Bug fixes
- [X] Styling
- [ ] Code Refactor

### Clubhouse Tickets Related:

- https://app.shortcut.com/terminalsystems/story/23851/fix-default-account-emoji

### Screenshots/Gifs:

Screenshot 2021-09-18 at 19 13 09

https://user-images.githubusercontent.com/32408893/133897412-ac0aa129-bc16-4e9a-b8a8-6aa69b80f234.mov

### Tested on:

- [X] Chrome
- [X] Firefox
- [ ] Safari

### CODE REVIEW

File: source/Options/Views/Welcome/steps/CreatePasswordStep.jsx [hunk 1]
```diff
@@ -51,8 +52,9 @@ const CreatePasswordStep = ({ handleNextStep, handleSetMnemonic, mnemonic }) =>
       apps: {},
     });
 
-    const type = mnemonic ? 'import-keyring' : 'create-keyring';
-    const params = { password, mnemonic };
+    const type = mnemonic ? HANDLER_TYPES.IMPORT : HANDLER_TYPES.CREATE;
```
Comment (rocky-fleek) [inline source/Options/Views/Welcome/steps/CreatePasswordStep.jsx:55]: THANK YOU lol I always forgot to add this 🤣

File: source/components/UserIcon/styles.js [hunk 2]
```diff
@@ -12,16 +12,18 @@ export default makeStyles({
...
     width: 60,
     height: 60,
-    fontSize: 34,
   },
   medium: {
     width: 40,
     height: 40,
-    fontSize: 22,
+    '& > span': {
+      transform: 'scale(0.7)',
```
Comment (rocky-fleek) [inline source/components/UserIcon/styles.js:25]: why sacle it down?

Comment (maksymboytsov) [reply source/components/UserIcon/styles.js:25]: Sidebar account's emoji wasn't placed correctly

**Review by rocky-fleek — APPROVED** [review]
Has conflicts but code's good

**Pull request merged**